modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/client/extension/NodeBuildConfigurer.java

```diff
@@ -77,21 +53,10 @@ public void apply(
 
 		_configureExtensionNode(nodeExtension);
 
-		TaskProvider<CreateClientExtensionConfigTask>
-			createClientExtensionConfigTaskProvider =
-				GradleUtil.getTaskProvider(
-					project,
-					ClientExtensionProjectConfigurator.
-						CREATE_CLIENT_EXTENSION_CONFIG_TASK_NAME,
-					CreateClientExtensionConfigTask.class);
-
-		createClientExtensionConfigTaskProvider.configure(
-			createClientExtensionConfigTask -> {
-				createClientExtensionConfigTask.dependsOn(
-					NodePlugin.PACKAGE_RUN_BUILD_TASK_NAME);
-				createClientExtensionConfigTask.doLast(
-					task -> _processAssembleIntructions(project));
-			});
+		assembleClientExtensionTaskProvider.configure(
```

Comment:
We should add `NodeConfigurer` to all client extension project.  Because this configurer will be a `no-op` if they don't have a package.json with build script.  I want to get rid of this:
https://github.com/liferay/liferay-portal/blob/master/modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java#L118-L132
and get rid of adding the NodeBuildConfigurer to multiple different "types".